docs/performance/transitions.md

```diff
@@ -107,7 +107,7 @@ The custom `sauce:performance` command measures the performance output against a
 
 |Argument|Description|
 |---|------|
-|`name`<br/><font size="2">Required</font>|A name of the test as it would appear in the Sauce Labs application.|
+|`name`<br/><font size="2">Required</font>|A name of the test as it would appear on Sauce Labs.|
```

Comment:
Curious why you use "on Sauce Labs" here? Seems like you say "in" for most of the other use cases.

Reply:
Fixed -- good catch, Nancy!